packages/cli-kit/assets/cli-ruby/lib/shopify_cli/theme/syncer.rb

```diff
@@ -47,6 +47,7 @@ def initialize(ctx, theme:, include_filter: nil, ignore_filter: nil, overwrite_j
         @error_reporter = ErrorReporter.new(ctx)
         @standard_reporter = StandardReporter.new(ctx)
         @reporters = [@error_reporter, @standard_reporter]
+        @notify = notify
```

Comment:
I believe we could remove the `@notify` from the `Syncer`. Am I missing something?